person.rb

```diff
@@ -1,26 +1,67 @@
-class Person
+class Nameable
+  def correct_name
+    raise NotImplementedError
+  end
+end
```

Comment:
- [ ] **[OPTIONAL]** I would recommend you move this class to its own file in keeping with Ruby conventions

Reply:
Done